frontend/src/Frontend/ChainwebApi.hs

```diff
@@ -105,6 +120,35 @@ getServerInfo nc = do
   esi <- getInfo (_netConfig_serviceHost nc <$ pb)
   holdDyn Nothing esi
 
+detailsXhr :: ChainwebHost -> ChainwebMeta -> Text -> Text -> Either String (XhrRequest ByteString)
+detailsXhr host meta token account = do
+    chainId <- note "Could not parse chain ID" $ chainIdFromText $ _chainwebMeta_chainId meta
+    let url = localUrl host chainId
+    let tx = mkTransaction pc []
+    pure $ XhrRequest "POST" url $ def
+      { _xhrRequestConfig_headers = "content-type" =: aj <> "accept" =: aj
+      , _xhrRequestConfig_sendData = BL.toStrict $ encode $ toJSON tx
+      }
+  where
+    aj = "application/json"
+    code = T.pack $ printf "(%s.details \"%s\")" token account
+    pc = PactCommand (ExecPayload $ Exec code Nothing) [] meta "local" Nothing
+
+transferXhr :: ChainwebHost -> Text -> Text -> Maybe Integer -> Maybe Int -> Maybe Int -> Maybe Int -> Maybe Text -> Either String (XhrRequest ())
+transferXhr ch account token chainid fromheight limit offset nextToken = do
+  let url = dataBaseUrl ch <> "/txs/account/" <> account <> if T.null rest then "" else "?" <> rest
+      rest = T.intercalate "&" $ ("token=" <> token) : catMaybes params
+      params =
+        [T.append "chain=" . T.pack . show <$> chainid
+	, T.append "fromheight=" . T.pack . show <$> fromheight
+	, T.append "limit=" . T.pack . show <$> limit
+	, T.append "offset=" . T.pack . show <$> offset
+        , T.append "next=" <$> nextToken
+	]
+  pure $ XhrRequest "GET" url $ def
+    { _xhrRequestConfig_responseHeaders = OnlyHeaders $ Set.singleton "Chainweb-Next"
+    }
```

Comment:
I think we should rewrite `transferXhr` to use `servant-reflex` like the clients for the other CW-D endpoints (such as `searchEvents` and `searchTxs` below.